src/js/providers/html5.js

```diff
@@ -756,7 +756,8 @@ define([
             var _selectedTextTrackIndex = -1, i = 0;
             _activeCuePosition = -1;
             for (i; i < _videotag.textTracks.length; i++) {
-                if (_videotag.textTracks[i].mode === 'showing') {
+                if ((_videotag.textTracks[i].kind === 'subtitles' || _videotag.textTracks[i].kind === 'captions')
+                    && _videotag.textTracks[i].mode === 'showing') {
```

Comment:
Putting && at the start of the line introduced a jshint error breaking the build. Fix that and put the track _videotag.textTracks[i] into a var when you need to get it 3 or more times.